libkbfs/modes.go

```diff
@@ -252,6 +261,12 @@ func (mso modeSingleOp) TLFEditHistoryEnabled() bool {
 	return false
 }
 
+func (mso modeSingleOp) SendEditNotificationsEnabled() bool {
+	// We don't want git, or other single op writes, showing up in the
+	// notification history.
```

Comment:
That said, long term this might be nice to consider.

Reply:
If you think we should keep them in the history, maybe with a special type in the notification so we can ignore them in the edit history.  I was worried this would flood the channel with irrelevant info and force us to scroll back through more messages than necessary to reconstruct the history.

It might be better in a completely different kbfs-edits channel, something like "<writer>:git" maybe.  In that case, we can probably do it at some later time if desired.

Reply:
Agreed, I was thinking something like a separate channel for later.